tools/nl/embeddings/build_embeddings.py

```diff
@@ -19,17 +19,22 @@
 import csv
```

Comment:
Given lack of e2e tests, could you try a medium_ft build to sanity check no regression?

Reply:
Ran medium_ft build and this is the validation diff:

https://storage.mtls.cloud.google.com/datcom-embedding-diffs/boxu_medium_ft_2024_03_04_10_19_20.html

There is one diff which is ok (two stat vars have same description and same score)